README.md

```diff
@@ -255,3 +311,27 @@ After that, you need to restart the apache server:
 ```
 systemctl restart apache2
 ```
+### Uninstalling Monasca UI plugin
+remove the symlinks:
+```
+rm <horizon_dir>/openstack_dashboard/enabled/_50_admin_add_monitoring_panel.py
+rm <horizon_dir>/openstack_dashboard/conf/monitoring_policy.json
+```
+Remove the line with the instruction to append site-packages in the wsgi file:
+
+Open file: `<horizon_dir>/openstack_dashboard/wsgi/django.wsgi`, delete the line: `sys.path.append("<monasca_ui_dir>/lib/python2.7/site-packages/")` and save.
+
+Delete created files: 
+```
+rm -rf <installation_dir>/monasca_ui
+```
+
+Delete the installer:
```

Comment:
Is it necessary to remove Horizon Plugin installer?

Reply:
Removed